config/config.exs

```diff
@@ -33,6 +33,14 @@ config :phoenix, :generators,
   migration: true,
   binary_id: false
 
+config :mime, :types, %{
+  "*/*" => ["json"],
+  "application/json" => ["json"],
+  "application/binary" => ["binary"],
+  "application/avro.binary" => ["avro"],
```

Comment:
If it's for the future, we add it in the future.
Having them now will lead to confusion.

And when it's avro.binary, there is no way for kastlex to know the schema ID.
when it's confluent.avro.binary, we need to have the schema registry URL in config or ?